CONTRIBUTE.md

```diff
@@ -7,49 +7,87 @@
+8. Commit the changes to the `CHANGELOG.md` file
+9. Write the PR number in the corresponding issue so that they are linked. Write it with one of the [special keywords](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) so that the issue will be automatically closed when the PR is merged (example: `Closes #<your issue number>`)
+10. [Ask](https://help.github.com/en/github/managing-your-work-on-github/assigning-issues-and-pull-requests-to-other-github-users) for review of your PR 
+ 
+    A) If including new terms to the ontology, ask at least two persons to review
+    
+    B) If restructuring the ontology, one person is enough, though two is better
+    
+    C) If your PR does not modify the ontology, only one reviewer is enough
+
+11. Check that, after this whole process, you branch does not have conflict with `dev` (github prevents you to merge if there are conflicts). In case of conflicts you are responsible to fix them on your branch before your merge (see below "Fixing merge conflicts" section)
+    
+12. (if approved) Merge the PR into `dev` and delete the branch on which you were working. In the merge message on github, you can notify people who are currently working on other branches that you just merged into dev, so they know they have to check for potential conflicts with `dev`
+   
    
-### Specific workflow
-We distinguish workflows between A) including new terms and B) restructuring existing parts.
+### Fixing merge conflicts
+
+
+Avoid large merge conflict by merging the updated `dev` versions in your branch.
 
-A) Including new terms to the ontology 
-- 1 PR for 1 new term (with maximum a few(!) subclasses, if at all)
-- each term has to include adequate bfo classification, proper definition, proper relations
-- a discussion on the topic from the energy modelling domain's point of view is recommended
-- review by at least 2 domain experts
+In case of conflicts between your branch and `dev` you must solve them locally.
 
-B) Restructuring existing parts: tbd.
+1. Get the latest version of `dev`
+    ```bash
+    git checkout dev
+    ```
+   
+    ```bash
+    git pull
+    ```
+    
+2. Switch to your branch
 
-Please include in the PR and issue name a [A] or [B] to indicate which part of the workflow you are refering to. 
+    ```bash
+    git checkout <your branch>
+    ```
+    
+3. Merge `dev` into your branch
+    ```bash
+    git merge dev
+    ```
+    
+4. The conflicts have to be [manually](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line) resolved
+    
 
-### Branch name convention
+### Conventions
 The convention is to always have `feature/` in the branch name. The `myfeature` part should describe shortly what the feature is about (separate words with `_`).
```

Comment:
does the `myfeature` part come after the slash? So the naming convention is `feature/myfeature` ?

Reply:
exactly :) 

This reminds me that @gnn suggested to have the issue number in the branch name so `feature/myfeature-#XY`, I will modify it accordingly

Reply:
Done in 55e48f691ba940e258c55d3e572fbd361a0acc9b